### PR TITLE
New package: lily-0.16

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/lily/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'lily'
+pkgname=lily
+version=0.16
+revision=1
+build_style=cmake
+hostmakedepends="cmake"
+short_desc="An interpreted language with a focus on expressiveness and type safety"
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="CUSTOM"
```
Comment (Duncaen) [inline srcpkgs/lily/template]: `MIT`

Comment (diogoleal) [inline srcpkgs/lily/template]: Changed

File: srcpkgs/lily/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'lily'
+pkgname=lily
+version=0.16
+revision=1
+build_style=cmake
+hostmakedepends="cmake"
```
Comment (ebfe) [inline srcpkgs/lily/template]: could be removed, the build_style already provides the host dependency on cmake now.

Comment (diogoleal) [inline srcpkgs/lily/template]: Removed.

**Pull request merged**